spec/mercure.md

```diff
@@ -592,11 +490,12 @@ least the following properties:
 *   `id`: the identifier of this update, it **MUST** be the same value as the subscription update's
     topic
 *   `type`: the fixed value `Subscription`
-*   `topic`: the topic selector used of this subscription
+*   `matchType`: the topic matcher type used of this subscription (**MUST** be omitted if the matcher type is exact matching)
```

Comment:
about what I said earlier, we should specify whether this is camel case, not case sensitive or lowercase

Reply:
done